lib/ruby-mpd/plugins/controls.rb

```diff
@@ -21,6 +21,7 @@ def pause=(toggle)
       def play(pos = nil)
         if pos.is_a?(Hash)
           if pos[:id]
+            # TODO What is priority?
             send_command :playid, priority, pos[:id]
           else
             raise ArgumentError, 'Only :id key is allowed!'
```

Comment:
It's a mistake in my refactoring, there's no `priority` argument. Line should be:

`send_command :playid, pos[:id]`